components/AdditionalResources.js

```diff
@@ -8,11 +8,17 @@ const AdditionalResources = () => (
     <div className="additional__resource">
       <span>Links, sources<br /> and emulators</span>
       <a href="http://webvr.info" target="_blank">wevbr.info</a>
```

Comment:
I'd recommend to not use `target="_blank"`, as it messes with expected link behaviours